test/functional/lua/ftplugin_spec.lua

```diff
@@ -0,0 +1,105 @@
+local t = require('test.testutil')
```

Comment:
this test file should probably live in `test/functional/editor/`. 

`lua/` tests are for the Lua stdlib. Not general features. The fact that this feature is implemented via `_ftplugin.lua` is an implementation detail.